tests/integration/targets/test_mysql_variables/tasks/assert_var_output.yml

```
@@ -35,6 +35,6 @@
 - name: assert output variable info match mysql variable info
   assert:
     that:
-      - "result.changed == true"
+      - "result.changed | bool == true"
```

```suggestion
      - "result.changed | bool == true"
```
i wonder if `result is changed` works here ? It must, i think, can we check?

---

Good point!

Mmh, I must recall how I made it fails in the first place. I think it was using Python 3.8 on stable-1 and target test_mysql_variables.

I let this PR open as a reminder to check that.